wp-content/themes/chrisdinh-photography/resources/views/sections/header.blade.php

```diff
@@ -1,11 +1,61 @@
-<header class="banner">
-  <a class="brand text-grey component" href="{{ home_url('/') }}">
-    {!! $siteName !!}
-  </a>
-
-  @if (has_nav_menu('primary_navigation'))
-    <nav class="nav-primary" aria-label="{{ wp_get_nav_menu_name('primary_navigation') }}">
-      {!! wp_nav_menu(['theme_location' => 'primary_navigation', 'menu_class' => 'nav', 'echo' => false]) !!}
-    </nav>
-  @endif
+<header class="banner relative">
+    <!-- Brand logo linking to the homepage -->
+    <a href="{{ home_url('/') }}" class="brand component font-logo text-3xl md:text-4xl lg:text-5xl text-white absolute top-8 left-10 md:top-20 md:left-24 z-10">
+        Chris D.
+    </a>
+
+    @if ($menuItems)
+    <!-- Circle SVG button for opening the navigation menu overlay -->
+        <button id="hamburger" class="hamburger-button absolute top-8 right-10 md:top-20 md:right-24 z-50" aria-label="Open menu">
+            <svg xmlns="http://www.w3.org/2000/svg" class="w-6 h-6" fill="white" viewBox="0 0 24 24" stroke="white">
+                <circle cx="12" cy="12" r="11" fill="white" />
+            </svg>
+        </button>
+
+        <!-- Full-screen menu overlay, hidden by default -->
+        <div id="menuOverlay" class="fixed inset-0 bg-neutral-900 opacity-0 hidden transition-opacity duration-175 ease-in-out z-40 bg-cover bg-center">
+            <!-- Container for the background image and its effects -->
+            <div id="backgroundImageContainer" class="fixed inset-0 bg-cover bg-center opacity-0 transition-opacity duration-1000 ease-in-out z-0"></div>
+                <div id="backgroundImageContainer2" class="fixed inset-0 bg-cover bg-center opacity-0 transition-opacity duration-1000 ease-in-out z-0"></div>
+                    <!-- Navigation Menu -->
+                    <div class="flex h-full flex-col-reverse lg:flex-row justify-between items-center z">
+                        <nav class="w-full flex flex-col justify-end items-end text-white lg:w-1/2 lg:justify-start lg:items-start lg:pl-48 z-20" aria-label="Primary Navigation">
+                            <ul class="flex flex-col w-full items-end pr-6 lg:items-start">
+                                @foreach ($menuItems as $menuItem)
+                                    <li class="nav-item mb-6" data-image="{{ $menuItem->navigation_image }}">
+                                        <a href="{{ $menuItem->url }}"
+                                        class="inline-block text-5xl lg:text-5xl transition-transform duration-300 ease-in-out hover:scale-110"
+                                        @if ($menuItem->link_target === '_blank') target="_blank" rel="noopener noreferrer" @endif>
```

Comment:
Setting a menu item in the admin to open in a new tab and then testing on the front end does not actually open the menu item in a new tab. There's a bug here, please fix

Reply:
added same functionality to instagram handle in commit 427aa9b6b39e2869116a72b7c2b1658270bdf950. please try with this new commit

Reply:
I see the issue here - going to resolve this comment and add a new comment @jonathanmdinh